rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -1030,14 +1030,14 @@ void main_window::SaveWindowState()
 
 void main_window::RepaintThumbnailIcons()
 {
+#ifdef _WIN32
 	const QColor new_color = gui::utils::get_label_color("thumbnail_icon_color");
 
 	const auto icon = [&new_color](const QString& path)
 	{
 		return gui::utils::get_colorized_icon(QPixmap::fromImage(gui::utils::get_opaque_image_area(path)), Qt::black, new_color);
 	};
 
-#ifdef _WIN32
```

Comment:
this is theoretically wrong, as the linux code should share the code above if ever implement

Reply:
They can just move ifdef, I don't see the problem.

Reply:
it was just a note

Reply:
Nvm, can use [[maybe_unused]] here as well. Thanks for suggestion.